### PR TITLE
9.x

### PR DESCRIPTION
**Pull request by veena-ux**

### CODE REVIEW

File: installation/docker-compose.adoc [hunk 1]
```diff
@@ -4,9 +4,9 @@ include::../partials/attributes.adoc[]
 {CANVAS} and its related services can be run on a single machine using `Docker Compose for BAMOE Canvas`.
 We can access the docker-compose file from the https://www.ibm.com/support/pages/node/7001673[Support and Download Page].
 
-== Prerequistes
+== Prerequisites
 
-You will need Docker and the Compose plugin in order to run the file. Refer to their web page for installation.
+You need the Docker and Compose plugins to run the file. See the web page for installation.
```
Comment (porcelli) [inline installation/docker-compose.adoc:9]: "the Docker"... feel not natural

**Review by porcelli — CHANGES_REQUESTED** [review]

File: installation/docker-compose.adoc (hunk 1 above)
Comment (LightGuard) [reply installation/docker-compose.adoc:9]: Ah.
You will need Docker and Docker Compose ... ?

Comment (LightGuard) [reply installation/docker-compose.adoc:9]: Take out the word "plugins"

Comment (porcelli) [reply installation/docker-compose.adoc:9]: "You'll need Docker and the Compose plugin" could work

Comment (LightGuard) [reply installation/docker-compose.adoc:9]: Or even:
You need Docker and Docker Compose to run the file.

Comment (porcelli) [reply installation/docker-compose.adoc:9]: works for me

**Review by LightGuard — APPROVED** [review]

**Pull request merged**